docs/readmes/cwf/dev_testing.md

```diff
@@ -64,33 +64,33 @@ Runs an iperf3 server to drive traffic through CWAG.
 To run all setup work and the entire CWF integration test suite, run
 
 ```bash
-[HOST] fab integ_test
+[HOST] fab integ-test
 ```
 
 Once the above command has been executed, which means that the set-up of the VMs etc. has been
-performed, command-line options can be utilized to rerun the tests without redoing the set-up
+performed, command-line options can be dropped to rerun the tests without redoing the set-up
```

Comment:
I would keep the `utilized` here, as we are using command line options, even if they are negatives.